_maps/shuttles/infiltrator_basic.dmm

```diff
@@ -1047,7 +1047,7 @@
 /obj/machinery/nuclearbomb/syndicate,
 /obj/machinery/door/window{
 	dir = 1;
-	name = "Theatre Stage"
+	name = "interior door"
```

Comment:
actually this is a dv.

you can just open this in notepad or w/e and remove the line, and the semicolon on the line right above it to get rid of it.